src/index.js

```diff
@@ -0,0 +1,87 @@
+window.wikiaJWPlayer = function (elementId, options, callback) {
+	/**
+	 * loads jwplayer library
+	 * @param elementId
+	 * @param playerURL
+	 * @param callback
+	 */
+	function loadJWPlayerScript(elementId, playerURL, callback) {
+		if (typeof jwplayer !== 'undefined') {
+			callback();
+			return;
+		}
+		var script = document.createElement('script'),
+			playerElement = document.getElementById(elementId);
+		script.onload = function () {
+			wikiaJWPlayerSettingsPlugin.register();
+			callback();
+		};
+		script.async = true;
+		script.src = playerURL || 'https://content.jwplatform.com/libraries/VXc5h4Tf.js';
+		// insert script node just after player element
+		playerElement.parentNode.insertBefore(script, playerElement.nextSibling);
```

Comment:
On News & Stories we have articles with multiple embedded videos. It doesn't seem to make sense to load the player script for every one. Can't it be loaded once, for all videos on the page?